src/query/sql/src/executor/format.rs

```diff
@@ -55,6 +55,63 @@ impl PhysicalPlan {
     ) -> Result<FormatTreeNode<String>> {
         to_format_tree(self, &metadata, &prof_span_set)
     }
+
+    pub fn format_join(
+        &self,
+        metadata: &MetadataRef,
+        prof_span_set: &ProfSpanSetRef,
+    ) -> Result<FormatTreeNode<String>> {
+        match self {
+            PhysicalPlan::TableScan(plan) => {
+                if plan.table_index == DUMMY_TABLE_INDEX {
+                    return Ok(FormatTreeNode::with_children(
+                        format!("Scan: dummy, rows: {}", plan.source.statistics.read_rows),
+                        vec![],
+                    ));
+                }
+                let table = metadata.read().table(plan.table_index).clone();
+                let table_name =
+                    format!("{}.{}.{}", table.catalog(), table.database(), table.name());
```

Comment:
Maybe only `table.name()` is enough, to make explain plan clearer

Reply:
it's already in one line, maybe there is same table name in different database.